db/migrate/20240322141829_add_resubmission_dates_to_prior_authority_application.rb

```diff
@@ -0,0 +1,6 @@
+class AddResubmissionDatesToPriorAuthorityApplication < ActiveRecord::Migration[7.1]
+  def change
+    add_column :prior_authority_applications, :resubmission_requested, :datetime
```

Comment:
I guess these may need to be on the `further_information[_request]s` associations, either as well or in addition to being here?! but for another PR/day